tests/test_udp.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 import sys
 
-from asyncio import test_utils
+from tests import utils as test_utils
```

Comment:
I don't like this "tests" import. Just move/copy the necessary functions to `uvloop._testbase`.